src/test/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/sidecarWorkingDir.groovy

```diff
@@ -0,0 +1,12 @@
+podTemplate(containers: [
+    containerTemplate(name: 'busybox', image: 'busybox', ttyEnabled: true, command: '/bin/cat', workingDir: '/src')
```

Comment:
Should the test assert something about `/src`?

Reply:
Right could be made slightly better.

Reply:
Done